Meeting_Recorde_180624.md

```diff
@@ -0,0 +1,26 @@
+Meeting record
+
+project name:
+   summary and plans for nba-tall project
+Date of meeting:
+   June 27,2018
+Location:
+    WeChat
+Minutes prepared by:
+  Wang Qing and JinYuxin
+
+Purpose of Meeting:
+  summary and plans for nba-tall and snails project
+Attendance at Meeting:
+  @MrKingHH @hxlong941110 @jsondrm @throcean @wangqing1218 @zmzhouXJTU @Tina66 @July-zhang
+
+Meeting Agenda:
+  Speak freely about the project subject
+Ation Items:
+   (1)Create aliyun server(@MrKingHH)
+	 (2)Finish the demos of **DIS** and **CS** of **nba-tall**.(@wangqing1218)
```

Comment:
Fix the format.

Reply:
Have fixed the format(the final commit).